testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/CLIEmbedServerTestCase.java

```diff
@@ -795,9 +798,9 @@ private void testPathDir(final String propName, final String value) throws IOExc
             for(String prop : SERVER_PROPS) {
                 if (prop.equals(propName)) {
                     assertPath(propName, ROOT + File.separator + newStandalone + File.separator + value);
-                } else {
-                    // just make sure the unchanged property has the default basedir
-                    //assertTrue(WildFlySecurityManager.getPropertyPrivileged(prop, "").indexOf(currBaseDir) != -1);
+//                } else {
```

Comment:
The intent here is to positively assert that an unchanged property hasn't been changed inadvertently, so has the default base.dir. Was this causing a failure somewhere?